src/main/java/org/openstreetmap/atlas/utilities/configuration/ConfiguredFilter.java

```diff
@@ -27,6 +30,13 @@
     public static final String DEFAULT = "default";
     public static final ConfiguredFilter NO_FILTER = new ConfiguredFilter();
     public static final String CONFIGURATION_ROOT = CONFIGURATION_GLOBAL + ".filters";
+    public static final String TYPE_JSON_PROPERTY_VALUE = "_filter";
+    public static final String NAME_JSON_PROPERTY = "name";
+    public static final String PREDICATE_JSON_PROPERTY = "predicate";
+    public static final String UNSAFE_PREDICATE_JSON_PROPERTY = "unsafePredicate";
```

Comment:
How about re-using those values in the private static values below?

Reply:
Haha I knew you would mention this. We can do that for now I suppose. I'd like the values to be public in case future code wants to parse the JSON and wants to know the key names. So maybe I can change those to public? Also, I still need to add unit tests...

Reply:
Yes, making all those public seems fine to me!